docs/src/main/asciidoc/native-and-ssl.adoc

```diff
@@ -244,11 +252,10 @@ quarkus.native.additional-build-args=-J-Djavax.net.ssl.trustStore=/tmp/mycerts,-
 ----
 
 will ensure that the certificates of `/tmp/mycerts` are baked into the native binary and used *in addition* to the default cacerts.
+The file containing the custom TrustStore does *not* (and probably should not) have to be present at runtime as its content has been baked into the native binary.
 
-[IMPORTANT]
-====
-The file containing the custom TrustStore does *not* have to be present at runtime as its content has been baked into the native binary.
-====
+Using the runtime certificate configuration, supported by GraalVM since `21.3` does not require any special or additional configuration compared to regular 
```

Comment:
and respectively another heading here:

```suggestion

=== Run time configuration

Using the runtime certificate configuration, supported by GraalVM since `21.3` does not require any special or additional configuration compared to regular 
```

This should make it easier for the reader to skip over build time configuration if they are not interested in it.

Reply:
Let's drop the backticks around the version. We don't backtick the versions.